uint8 joyX
uint8 joyY
bool buttonZ
bool buttonC
float32 accelX
float32 accelY
float32 accelZ